packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +813,354 @@ components:
+                example: 1080
+              frameRate:
+                type: number
+                description: Frame rate of the track - only for video tracks
+                example: 30
+              id:
+                type: string
+                description: ID of the track
+                example: ABC123
+              duration:
+                type: number
+                description: Duration of the track in seconds
+                example: 23.8238
+              bitrate:
+                type: number
+                description: Bitrate of the track in bits per second
+                example: 1000000
+              channels:
+                type: number
+                description: Amount of audio channels in the track
+                example: 2
+              sampleRate:
+                type: number
+                description:
+                  Sample rate of the track in samples per second - only for
+                  audio tracks
+                example: 44100
+              bitDepth:
+                type: number
+                description: Bit depth of the track - only for audio tracks
+                example: 16
+              codec:
+                type: string
+                description: Codec of the track
+                example: aac
+        profiles:
+          type: object
+          description: HLS Profile manifests associated with the asset
+          properties:
+            kind:
+              type: string
+              description: Type of the profile
+              example: manifest
+            name:
+              type: string
+              description: Filename of the manifest
+              example: variant_1080.m3u8
+            width:
+              type: number
+              description: width of the profile
+              example: 1920
+            height:
+              type: number
+              description: height of the profile
+              example: 1080
+            frameRate:
+              type: number
+              description: Frame rate of the profile
+              example: 30
+            id:
+              type: string
+              description: ID of the profile
+              example: ABC123
+            duration:
+              type: number
+              description: Duration of the profile in seconds
+              example: 23.8238
+            bitrate:
+              type: number
+              description: Bitrate of the profile in bits per second
+              example: 1000000
+        playbackUrl:
+          type: string
+          description: Playback url of the playlist manifest
+          example: https://cdn.livepeer.com/ABC123/playlist.m3u8
+        status:
+          type: string
+          description: Status of the asset
+          example: ready
+        originTaskId:
+          type: string
+          description: Task ID of the task that created the asset
+          example: 1-23-123-123-1123-123
+        relatedTasks:
```

Comment:
I don't know if it's better to have the `relatedTasks` here in the Asset object to have a view of the task related to it, or if we can display them just with an `assetId` param in the task object, wdyt?

Reply:
I think we should have unidirectional links, of children pointing to the parent. So the tasks should have 1 indexed field for the source asset, and if we want to list all related (children) tasks we just query the task table.

Reply:
I agree to remove this field, unidirectional links are more than enough